docs/Dialects/FIRRTL/FIRRTLIntrinsics.md

```diff
@@ -289,3 +289,94 @@ always @(posedge clock) begin
   end
 end
 ```
+
+### circt_view
+
+This will become a SystemVerilog Interface that is driven by its arguments.
+This is _not_ a true SystemVerilog Interface, it is only lowered to one.
+
+
+| Parameter     | Type   | Description                                         |
+| ------------- | ------ | --------------------------------------------------- |
+| name          | string | Instance name of the view.                          |
+| view          | string | JSON encoding the view structure.                   |
+
+| Port       | Direction | Type     | Description                         |
+| ---------- | --------- | -------- | ----------------------------------- |
+| ...        | input     | Ground   | Leaf ground values for the view     |
```

Comment:
This "port" and "direction" language is confusing to me. However, it is what the intrinsics do, so out of scope. 🙃

Reply:
These should be organized/formatted differently, I think this is holdover from intmodule days.

So not for this PR, but agreed :+1:.